test/integration/tas/tas_test.go

```diff
@@ -567,6 +521,159 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 			})
 		})
 
+		ginkgo.When("Nodes are created before test with the hostname being the lowest level", func() {
```

Comment:
Actually, I tested locally that the test passes after https://github.com/kubernetes-sigs/kueue/pull/3694, which is not good. 

The key point is that if we apply the skip form the PR above, then the usage coming from admitted workloads is ignored. So, we can exploit it by creating second workload which would not fit, but fits if the usage is ignored.
One way to do it: don't delete `wl1` and bump the count of pods for `wl2` to 4. After this PR wl2 should be inadmissible, but after the skip usage wl2 would get admitted.

Reply:
Fixed